### DISCUSSION TITLE
Use of mask

### DISCUSSION THREAD

**Discussion by roaldarbol** [mooch443/trex#84] [Ideas & Feedback]
This is not really a recommendation as much as a question. I've noticed that there is a parameter called `video_mask`, and when generating the `.pv`, there are mentions of `no mask` in the command line output. However, it doesn't appear in the documentation (yet, I'm sure).
So I was wondering how one could make use of a masking feature? In my videos, it could be quite useful to use a mask layer with a few circles (see attached image), so if you could shed some light on it that would be greatly helpful! Maybe also a pointer on how to create a mask. 
Thanks!
![tracking_ir_background](https://user-images.githubusercontent.com/25629697/116566304-8d41ee80-a8fe-11eb-803e-2a5a80cdcb3a.png)

**Comment by mooch443** [comment]
Hey, the video_mask is supposed to be an actual video that is to be used as a mask - of which all the white parts are immediately parsed as individuals. This is designed to be used when segmenting individuals with external software, but you still want to keep the greyscale values. Internally, TGrabs supports `cam_circle_mask` which just puts one circle in the center of the screen (with max dimensions). So far, though, I did not implement any way of customizing this. If this is important, I may be able to include it with the next version.

However, I doubt that it is. TRex provides `track_include` which can contain an array of shapes (selectable by holding strg/CMD+clicking) where you could approximate the circles. This can be part of a .settings file, of course, afterwards. Ask away if you need more clarification. And thanks for reporting the inconvenience! I'll update some texts.

**Comment by roaldarbol** [comment]
Hi Tristan, currently it is not super important or urgent. But maybe consider it; it enables some parallelization of experiments without adding any hardware which I think would be useful.

I'll have a go at `track_include` and (maybe) get back to you with questions. And no inconvenience at all! Just curiously digging away at your awesome work ;-)
